File: 3.2/java/SumOfRange.java
import java.util.Arrays;
import java.util.Scanner;

public class SumOfRange {
    public static void main(String[] args) {

        // create Scanner object
        Scanner in = new Scanner(System.in);

        System.out.print("Set number: ");
        // trying to parse input as an int
        // n is declared and initialized
        int n = in.nextInt();

        // make array with 'n' empty elements
        int[] range = new int[n];

        int sum = 0;
      
        // set array interating 'for' loop
        for (int i = 0; i < range.length; i++) {
            range[i] = i+1;
        }

        // print Nums (as array), Sum and 'newline'
        System.out.println("Nums: " + Arrays.toString(range));
        System.out.printf("Sum: %d\n", Arrays.stream(range).sum());
    }
}

File: 3.2/c/sum_of_range.c
#include <stdio.h>

// main() is standard entrypoint
// int is return type
int main()
{
    // n is declared but not initialized
    int n;

    printf("Set number: ");

    // parse input as int and place at n's address
    scanf("%d", &n);

    int sum = 0;

    // set and print array interating 'for' loop
    printf("Nums: [");
    for (int i = 0; i < n; i++)
    {
        // sum = sum + <next num>
        sum += i+1;

        // print elements on each iteration
        printf("%d", i+1);

        // suffix depends on iteration
        if (i == n-1)
            printf("]\n");
        else
            printf(", ");
    }

    // print Sum and 'newline'
    printf("Sum: %d\n", sum);

    // must return an int (0 on succes by default)
    return 0;
}

